clio-server/src/main/scala/org/broadinstitute/clio/server/webservice/IndexWebService.scala

```diff
@@ -22,19 +24,26 @@ abstract class IndexWebService[CI <: ClioIndex](
   private[webservice] def pathPrefixKey: Directive1[indexService.clioIndex.KeyType]
 
   private[webservice] val postMetadata: Route = {
-    pathPrefix("metadata") {
-      pathPrefixKey { key =>
-        post {
-          entity(as[indexService.clioIndex.MetadataType]) { metadata =>
-            complete(indexService.upsertMetadata(key, metadata))
+    implicit val jsonStreamingSupport: JsonEntityStreamingSupport =
+      JsonWebService.singleElememtJsonStreamingSupport
+    pathPrefix(metadataString) {
+      parameter(Symbol(forceString).as[Boolean] ? false) { force =>
+        pathPrefixKey { key =>
+          post {
+            entity(as[indexService.clioIndex.MetadataType]) { metadata =>
+              complete(
+                indexService
+                  .upsertMetadata(key, metadata, force)
```

Comment:
We can make this return a strict entity to avoid downstream pain with something like:
```scala
private[webservice] val postMetadata: Route = {
    pathPrefix(metadataString) {
      parameter(Symbol(forceString).as[Boolean] ? false) { force =>
        pathPrefixKey { key =>
          post {
            entity(as[indexService.clioIndex.MetadataType]) { metadata =>
              extractRequestContext { ctx =>
                import ctx.materializer

                complete {
                  // Type inference failed for me just trying to return this,
                  // but maybe we can massage it in a different way
                  val fut = indexService
                    .upsertMetadata(key, metadata, force)
                    .runWith(Sink.head[UpsertId])
                  fut
                }
              }
            }
          }
        }
      }
    }
  }
```

Reply:
Then we don't need the special-case `JsonEntityStreamingSupport`, either